### DISCUSSION TITLE
Yolov5 tuning doubt

### DISCUSSION THREAD

**Discussion by unrue** [ultralytics/yolov5#13045] [General]
I'm using Yolov5 and I have a doubt regarding evolve phase. The evolve phase, should be done before the final training or after?

I mean, through the evolve phase I'm able to find best hyper-parameters. Such parameters, should be used to make a final training, or not?

In evolve directory, I don't see anything like weights/best.pt:

`results.csv
results.png
hyp_evolve.yaml
evolve.csv
labels_correlogram.jpg
train_batch1.jpg
train_batch0.jpg
labels.jpg`

So, I suppose that once the evolve phase is finished, I'm able to train my net using best hyperparameter possible with hyp_evolve.yaml. Right? Thanks.
